sigstore-testkit/src/main/java/dev/sigstore/testkit/tuf/TestResources.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.testkit.tuf;
+
+import com.google.common.io.Resources;
+import dev.sigstore.json.GsonSupplier;
+import dev.sigstore.tuf.model.Root;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class TestResources {
+
+  public static final Path CLIENT_TRUSTED_ROOT = Path.of(Resources.getResource("dev/sigstore/tuf/trusted-root.json").getPath());
```

Comment:
I wonder if that works in case the resource comes from a jar

Reply:
:shrug: Should we cross that bridge when/if we get to it?

Reply:
I'm fine if it passes tests. I just got bitten multiple times by the fact that "resource loaded from jar does not have proper file path"

Reply:
I hear you. In this case it's just test code I think we're good. (famous last words) 

I think you need to click the approve button.